### PR TITLE
feature(logging): replaced bunyan logger with external winston

### PR DESCRIPTION
**Pull request by Pyrolistical**
kept default console logger when none is provided
fixed bug where cache was not being cleared

### CODE REVIEW

**Comment by shanejonas** [comment]
:+1:

File: lib/apicache.js [hunk 1]
```diff
@@ -41,20 +37,25 @@ function cacheDriver() {
...
     client.sadd(group, key, callback);
-  }
+  };
 
   this.delGroup = function(group, callback) {
-    log.debug('deleting group %s', group);
+    log.verbose('deleting group %s', group);
     group = prefixKey('group:' + group);
+    var _this = this;
     client.smembers(group, function(err, results) {
```
Comment (shanejonas) [inline lib/apicache.js:57]: this doesnt handle the error

**Comment by jbueza** [comment]
:+1:

**Comment by shanejonas** [comment]
:+1:

File: lib/apicache.js (hunk 1 above)
Comment (Pyrolistical) [inline lib/apicache.js:57]: fixed

**Pull request merged**